src/Glpi/Inventory/Asset/NetworkPort.php

```diff
@@ -335,8 +338,7 @@ private function handleMacConnection(\stdClass $port, int $netports_id)
             return;
         }
 
-        //reset, will be populated from rulepassed
-        $this->connection_ports = [];
```

Comment:
Most of the changes have a low impact. This one is more complex; phpstan did not understand it's populated from the rulepassed method.
I've tried to move the reset in the rulepassed to avoid phpstan issue; we'll see if tests are OK.

Reply:
Not a surprise; this breaks tests.